contracts/oss.yml

```diff
@@ -5590,6 +5590,42 @@ paths:
         default:
           description: Unexpected error
           $ref: '#/components/responses/ServerError'
+  /legacy/authorizations:
+    servers:
+      - url: /private
+    $ref: '#/paths/~1authorizations'
```

Comment:
I'm surprised these are showing up in the "normal" `oss.yml` contract... I think I know a way to fix it, but it'll require some copy-paste.